### ISSUE TITLE
I can define use reserved keywords as identifers

### ISSUE THREAD

**Issue by Maltysen** [cheddar-lang/Cheddar#41]
```
cheddar> var var = 6
cheddar> var
 6
cheddar> var = 7
 7
cheddar> var
 7
cheddar> var*6
 42
```

**Comment by vihanb** [comment]
But it works so there is no bug here

**Comment by Maltysen** [comment]
actually, this extends to all keywords

```
cheddar> var print = 123
cheddar> print
 123
```

**Comment by Maltysen** [comment]
but it *should* error

**Comment by vihanb** [comment]
okay that's a problem.

**Comment by vihanb** [comment]
wait why isn't it erroring... it should... ;_;

**Comment by vihanb** [comment]
Fixed in 7f061dfa663060d8a8517483a8cac8b4be2f781e
